api/src/main/java/com/creatubbles/api/model/user/avatar/Avatar.java

```diff
@@ -0,0 +1,109 @@
+package com.creatubbles.api.model.user.avatar;
+
+import android.support.annotation.Nullable;
+
+import com.creatubbles.api.model.creation.Creation;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.jasminb.jsonapi.annotations.Id;
+import com.github.jasminb.jsonapi.annotations.Relationship;
+import com.github.jasminb.jsonapi.annotations.Type;
+
+import java.util.Date;
+
+/**
+ * Created by Janek on 24.10.2016.
+ */
+@Type("user_avatars")
+public class Avatar {
+
+    @JsonCreator
+    public Avatar() {
+    }
+
+    public Avatar(@Nullable AvatarSuggestion avatarSuggestion, Creation avatarCreation) {
```

Comment:
1. This constructor can be private.
2. Why suggestion is nullable but when using builder it's required to provide suggestion (it's in the constructor of builder)?